File: App 5/app/src/main/java/y/app_3/aula6/Tela2.java
package y.app_3.aula6;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class Tela2 extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.tela_2);
        Button buto2 = findViewById(R.id.buto2);

        buto2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(Tela2.this, MainActivity.class);
                startActivity(i);
            }
        });
    }

}

File: App 1/app/src/main/java/y/app_1/Tela2.kt
package y.app_1

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity;
import kotlinx.android.synthetic.main.page_2.*

class Tela2: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?){
        super.onCreate(savedInstanceState)
        setContentView(R.layout.page_2)

        returnButton.setOnClickListener {
            startActivity(Intent(this,MainActivity::class.java))
        }
    }
}